docs/doxygen/ie_docs.xml

```diff
@@ -311,6 +311,75 @@ limitations under the License.
                 </tab>
                 <tab type="usergroup" title="Using Low-Precision 8-bit Integer Inference" url="@ref openvino_docs_IE_DG_Int8Inference">
                 </tab>
+                <tab type="usergroup" title="Low Precision Transformations" url="@ref openvino_docs_IE_DG_lpt">
```

Comment:
really? 😄 
Do we need LPT as first class citizen in public docs?

Reply:
We need it here https://docs.openvinotoolkit.org/latest/ie_plugin_api/index.html

In Plugin Development guide near IE Transformations

Reply:
done, thanks